### ISSUE TITLE
Exception in Parsing??

### ISSUE THREAD

**Issue by ItSeldomRains** [carpedm20/fbchat#469]
## Description of the problem
Copy pasted code from examples in the documentation, code fails to run and outputs the error shown below.
## Code
```
# -*- coding: UTF-8 -*-


from fbchat import log, Client


class EchoBot(Client):
    def onMessage(self, author_id, message_object, thread_id, thread_type, **kwargs):
        self.markAsDelivered(thread_id, message_object.uid)
        self.markAsRead(thread_id)

        log.info("{} from {} in {}".format(message_object, thread_id, thread_type.name))

        
        if author_id != self.uid:
            self.send(message_object, thread_id=thread_id, thread_type=thread_type)


client = EchoBot("email", "password")
client.listen()
```

(this code is exactly from the documentation)

```
When run, it shows such error

Traceback (most recent call last):
(note: i changed some numbers idk if it is relevant, tell me if it is)
Listening...
Exception in parsing of {u'queue': randomnumbershehe, u'ofd_ts': randomnumbersheheL,
u'iseq': 1070, u'type': u'delta', u'delta': {u'body': u'zeit', u'irisSeqId': u'1
070', u'attachments': [], u'requestContext': {u'apiArgs': {}}, u'messageMetadata
': {u'threadReadStateEffect': u'MARK_UNREAD', u'timestamp': u'1567352763886', u'
tags': [u'source:chat:web'], u'actorFbId': u'myfacebookID', u'skipSnippetUpda
te': False, u'offlineThreadingId': u'6573953959226412311', u'messageId': u'mid.$
cAAAAC517kHpzHsZ97ls7YEutdo0X', u'folderId': {u'systemFolderId': u'INBOX'}, u'th
readKey': {u'otherUserFbId': u'myfacebookID'}, u'skipBumpThread': False}, u'c
lass': u'NewMessage', u'irisTags': []}}
Traceback (most recent call last):
  File "C:\Python27\lib\site-packages\fbchat\_client.py", line 2775, in _parseMe
ssage
    self._parseDelta(m)
  File "C:\Python27\lib\site-packages\fbchat\_client.py", line 2739, in _parseDe
lta
    timestamp=ts,
  File "C:\Python27\lib\site-packages\fbchat\_message.py", line 373, in _from_pu
ll
    rtn.emoji_size = EmojiSize._from_tags(tags)
  File "C:\Python27\lib\site-packages\fbchat\_message.py", line 29, in _from_tag
s
    data = tag.split(":", maxsplit=1)
TypeError: split() takes no keyword arguments

## Environment information
- Python 2.7 version
- `fbchat` v1.8.1 version


If you have done any research, include that.
Make sure to redact all personal information.

**Comment by gave92** [comment]
I get the same error when using python 2.7. Switching to python 3.x solves the problem for me.
The fix would be editing "C:\Python27\lib\site-packages\fbchat\_message.py", line 29
from `data = tag.split(":", maxsplit=1)` to this `data = tag.split(":", 1)`

**Comment by madsmtm** [comment]
This is solved in `v1.8.2`, 6b68916d74df483bc5e1fbe12c2082f39554bbc8, using your suggested fix, @gave92 👍
